pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,19 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+In HTML, information that is programmatically determinable from a link includes [text nodes][] that are [included in the accessibility tree][] with a non-empty ("") [accessible name][] and are
+
+- in the same paragraph (HTML `p` element) as the link;
+- in the same list (HTML element with the semantic role of `listitem`) as the link;
+- in the same table cell (HTML element with the semantic role of `cell` or `gridcell`) as the link;
+- in a table header cell (HTML element with the semantic role of `columnheader` or `rowheader`) that is [associated][] with the table cell (HTML element with the semantic role of `cell` or `gridcell`) of which the link is a descendent;
```

Comment:
Descendent in the flat tree I'm guessing. You may want to create a definition of "descendent" that specifies it's a descendent in the flat tree.

Reply:
Updated, but I don't think the definition is needed